docs/api.rst

```diff
@@ -0,0 +1,339 @@
+#################
+API Documentation
+#################
+
+.. _http-apis:
+
+GET /
+=====
+
+The returned value is a JSON mapping containing:
+
+- ``hello``: the name of the service (``"reading list"``)
+- ``version``: complete version (``"X.Y.Z"``)
+- ``url``: absolute URI (without a trailing slash) of the API (*can be used by client to build URIs*)
+- ``eos``: date of end of support in ISO 8601 format (``"yyyy-mm-dd"``, undefined if unknown)
+- ``documentation``: The url to the service documentation. (What you are reading!)
+
+
+GET /__heartbeat__
+==================
+
+Return the status of each service the *reading list* depends on. The
+returned value is a JSON mapping containing:
+
+- ``database`` true if operational
+
+Return ``200`` if the connection with each service is working properly
+and ``503`` if something doesn't work.
+
```

Comment:
What do you think if we put these two *generic* endpoints in a separate doc file ?